azurerm/internal/services/digitaltwins/parse/digital_twins_endpoint_test.go

```diff
@@ -0,0 +1,128 @@
+package parse
```

Comment:
same above

Reply:
Since it is auto generated we should prolly leave it alone for now. I will speak to Tom about how to go about maybe exposing a naming override since in some cases the name may not make sense or be unnecessarily long.